pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -161,34 +162,38 @@ func skipRevisionedInOnDiskLocation(location certgraphapi.OnDiskLocation) bool {
 	return false
 }
 
-func RewriteNodeIPs(nodeList []*corev1.Node) *metadataOptions {
-	nodes := map[string]int{}
+func rewriteSecretDetails(secret *corev1.Secret, original, replacement string) {
+	name := strings.ReplaceAll(secret.Name, original, replacement)
+	if secret.Name != name {
+		secret.Name = name
+		if len(secret.Annotations) == 0 {
+			secret.Annotations = map[string]string{}
+		}
+		// Replace node name from annotation value
+		for key, value := range secret.Annotations {
```

Comment:
ok for now, but I suspect we may need to do keys too.

Reply:
Right, agreed